docs/serving/autoscaling/scale-bounds.md

```diff
@@ -71,4 +73,60 @@ spec:
 {{< /tab >}}
 {{< /tabs >}}
 
+## Initial scale
+
+This value controls the initial target scale of a revision after creation. If lower bound is set and is greater than the initial scale, the initial scale will be ignored.
```

Comment:
I think this could use an extra sentence to describe what initial scale actually does. Maybe something like:

```suggestion
This value controls the initial target scale of a revision after creation. This is the target scale a revision must reach, immediately after first creation, before being marked Ready. Once the revision has reached this scale once, this value is ignored.

If a lower bound is set, the revision must achieve this scale before being marked Ready, so if it is greater than the initial scale, the initial scale will effectively be ignored.
```

Reply:
Yes this would be very helpful. Should we also mention the revision will be scale down immediately after the initial target scale is reached if the received traffic only needs a smaller scale?

Reply:
Depends if we think "Once the revision has reached this scale once, this value is ignored" is enough for readers to guess this.. I think it might be, but also I could see any extra sentence being helpful to spell it out.